README.md

```diff
@@ -68,24 +66,39 @@ dokku postgres:link lolipop playground
 
 # the above will expose the following environment variables
 #
-#   DATABASE_URL=postgres://postgres:SOME_PASSWORD@172.17.0.1:5432/lolipop
-#   DATABASE_NAME=/lolipop/DATABASE
-#   DATABASE_PORT=tcp://172.17.0.1:5432
-#   DATABASE_PORT_5432_TCP=tcp://172.17.0.1:5432
-#   DATABASE_PORT_5432_TCP_PROTO=tcp
-#   DATABASE_PORT_5432_TCP_PORT=5432
-#   DATABASE_PORT_5432_TCP_ADDR=172.17.0.1
-
-# you can examine the environment variables
-# using our 'playground' app's env command
-dokku run playground env
-
-# you can customize the environment
-# variables through a custom docker link alias
-dokku postgres:alias lolipop POSTGRES_DATABASE
+#   DOKKU_POSTGRES_LOLIPOP_NAME=/lolipop/DATABASE
+#   DOKKU_POSTGRES_LOLIPOP_PORT=tcp://172.17.0.1:5432
+#   DOKKU_POSTGRES_LOLIPOP_PORT_5432_TCP=tcp://172.17.0.1:5432
+#   DOKKU_POSTGRES_LOLIPOP_PORT_5432_TCP_PROTO=tcp
+#   DOKKU_POSTGRES_LOLIPOP_PORT_5432_TCP_PORT=5432
+#   DOKKU_POSTGRES_LOLIPOP_PORT_5432_TCP_ADDR=172.17.0.1
+#
+# and the following will be set on the linked application by default
+#
+#   DATABASE_URL=postgres://postgres:SOME_PASSWORD@dokku-postgres-lolipop:5432/lolipop
 
-# you can also unlink a postgres service
+# another service can be linked to your app
+dokku postgres:link other_service playground
+
+# since DATABASE_URL is already in use, another environment variable will be
+# generated automatically
+#
+#   DOKKU_POSTGRES_BLUE_URL=postgres://postgres:ANOTHER_PASSWORD@dokku-postgres-other_service:5432/other_service
```

Comment:
Not sure what you mean but when linking the service to an app it will set `DATABASE_URL` if it’s available.